.github/workflows/announce-release.yaml

```diff
@@ -64,25 +65,31 @@ jobs:
           fi
           export OUTPUT_CHANNELS=${slack_channels[*]}
           export OUTPUT_JSON=$(echo $OUTPUT_CHANNELS | jq -R 'split(" ")')
-          echo "$OUTPUT_JSON" >> "$GITHUB_OUTPUT"
+          echo "channels=$OUTPUT_JSON" >> "$GITHUB_OUTPUT"
   
   get_announcement:
     name: Get announcement
     runs-on: ubuntu-latest
     outputs:
-      announcement: ${{ steps.get_announcement.outputs.announcement }}
+      announcement: ${{ steps.get_announcement_md.outputs.announcement }}
     steps:
       - name: Get announcement.md
         id: get_announcement_md
         run: |
-          export LINK="https://raw.githubusercontent.com/giantswarm/releases/refs/heads/master/${{ inputs.provider }}/${{ inputs.version }}/announcement.md"
+          PROVIDER=${{ inputs.provider }}
+          if [[ "$PROVIDER" == "capz" ]]; then
```

Comment:
if selecting capz as the provider we must convert it to azure to find the right directory path